logging/logging.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2022 Fraunhofer AISEC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//           $$\                           $$\ $$\   $$\
+//           $$ |                          $$ |\__|  $$ |
+//  $$$$$$$\ $$ | $$$$$$\  $$\   $$\  $$$$$$$ |$$\ $$$$$$\    $$$$$$\   $$$$$$\
+// $$  _____|$$ |$$  __$$\ $$ |  $$ |$$  __$$ |$$ |\_$$  _|  $$  __$$\ $$  __$$\
+// $$ /      $$ |$$ /  $$ |$$ |  $$ |$$ /  $$ |$$ |  $$ |    $$ /  $$ |$$ | \__|
+// $$ |      $$ |$$ |  $$ |$$ |  $$ |$$ |  $$ |$$ |  $$ |$$\ $$ |  $$ |$$ |
+// \$$$$$$\  $$ |\$$$$$   |\$$$$$   |\$$$$$$  |$$ |  \$$$   |\$$$$$   |$$ |
+//  \_______|\__| \______/  \______/  \_______|\__|   \____/  \______/ \__|
+//
+// This file is part of Clouditor Community Edition.
+
+package logging
+
+import (
+	"fmt"
+
+	"clouditor.io/clouditor/api/orchestrator"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	LoglevelDebug = "debug"
+	LoglevelInfo  = "info"
+	LoglevelError = "error"
+	Create        = "created"
+	Remove        = "removed"
```

Comment:
There are already constants for Read, Update, Delete, etc. in the `api` (or service?) package related to the authorisation strategy. We can/should re-use them instead of new ones I think.

In general, I would avoid string-type constants, you can just use Go enums.

Reply:
I used strings because I use them again in the log message `(fmt.Sprintf("%s %s with ID '%s'", req.GetType(), request, req.GetPayloadID())).` I wanted to avoid writing more ifs for the requestType. I am not sure how to use enums with corresponding strings.

Reply:
I used strings because I use them again in the log message `(fmt.Sprintf("%s %s with ID '%s'", req.GetType(), request, req.GetPayloadID())).` I wanted to avoid writing more ifs for the requestType. I am not sure how to use enums with the corresponding strings. But I will find a solution.